dolphinscheduler-extract/dolphinscheduler-extract-base/src/main/java/org/apache/dolphinscheduler/extract/base/client/NettyRemotingClient.java

```diff
@@ -166,13 +166,10 @@ private Channel getOrCreateChannel(Host host) {
      * @param host host
      * @return channel
      */
-    private Channel createChannel(Host host) {
+    Channel createChannel(Host host) {
         try {
-            ChannelFuture future;
-            synchronized (bootstrap) {
```

Comment:
why remove `synchronized` here?

Reply:
No need to lock when we want to create a channel.